### ISSUE TITLE
Use css calc function

### ISSUE THREAD

**Issue by LucidityDesign** [felix-horn/trump-quotes#12]
Ich glaube sowas wie
```
height: calc(100vh - 130px)
```
müsste auch funktionieren.

https://github.com/felix-horn/trump-quotes/blob/0aee780afb148f924d33d1b8d5981974d85b9bc6/src/PageQuote.js#L43

**Comment by felix-horn** [comment]
Danke für das Feedback auch über diesen Kanal :))) An dem Teil hab' ich echt lange rumprobiert und war bis zum Schluss nicht ganz glücklich. Mit `100vh` habe ich es zum Anfang probiert, das blöde ist nur, dass das mobil z.B. die Adresszeile vom Browser mit einschließt und deshalb der Body der eigentlichen App nicht `100vh` ist... `window.innerHeight` ist genau das, aber ich konnte es nicht im Styling verwenden (zumindest wusste ich nicht, wie) - daher habe ich die const außerhalb definiert... :)

**Comment by LucidityDesign** [comment]
Ja, das ist ein großes Problem, für das es keine CSS Lösung gibt:
https://stackoverflow.com/questions/37112218/css3-100vh-not-constant-in-mobile-browser

`height: -webkit-fill-available;` kannst du noch ausprobieren.

Ansonsten hilft nur ein resize-listener, der die Höhe setzt, sobald sich die Viewport-Größe verändert:

```
window.addEventListener('resize', () => ….)
```
